stats/src/main/java/com/facebook/airlift/stats/cardinality/SparseHll.java

```diff
@@ -171,7 +171,7 @@ public void eachBucket(BucketListener listener)
             // if zeros > EXTENDED_BITS_LENGTH - indexBits, it means all those bits were zeros,
             // so look at the entry value, which contains the number of leading 0 *after* EXTENDED_BITS_LENGTH
             int bits = EXTENDED_PREFIX_BITS - indexBitLength;
-            if (zeros > bits) {
+            if (zeros >= bits) {
```

Comment:
This is a correctness fix.  We should call this out explicitly in the commit message

Reply:
Good point, I'll put :
"Improved accuracy and performance of hyperloglog functions"